### PR TITLE
account_invoice_update_wizard: improve UI

### PR DESCRIPTION
**Pull request by sebastienbeau**
@alexis-via @khoivo1601 UX improvement and add posibillity to edit the account_id

### CODE REVIEW

**Comment by sebastienbeau** [comment]
Note: I have remove the edition of payment term as it's broken, I do not need it yet so at least we can hide it for now and fix it later

File: account_invoice_update_wizard/wizard/account_move_update.py [hunk 1]
```diff
@@ -251,3 +254,9 @@ class AccountMoveLineUpdate(models.TransientModel):
         'account.analytic.account', string='Analytic Account')
     analytic_tag_ids = fields.Many2many(
         'account.analytic.tag', string='Analytic Tags')
+    # Note : in france account_id is not a legal field of the invoice
+    # so we should be able to update it
```
Comment (alexis-via) [inline account_invoice_update_wizard/wizard/account_move_update.py]: ??? I don't understand how you can say "account_id is not a legal field of the invoice" ! The goal of this module is to be able to update only non-legal fields such as the analytic account on the invoice lines. But account_id is clearly a legal field.

Comment (sebastienbeau) [reply account_invoice_update_wizard/wizard/account_move_update.py]: I mean on an invoice (the pdf) you do not definie show the account_id, so if you did affect the right account id you can change it, it doesn't change anything for the pdf file

**Comment by sebastienbeau** [comment]
@alexis-via can you review, it's ready

**Review by alexis-via — APPROVED** [review]

**Pull request merged**